### PR TITLE
Rename _fetchEntity to _fetchEntityWithToken and add a plain old _fet…

### PR DESCRIPTION
**Pull request by CodeBaboon**
…chEntity

### CODE REVIEW

**Comment by ChrisKraljevicD2L** [comment]
|+60 -9277
Hachi machi

File: d2l-fetch-siren-entity-behavior.html [hunk 1]
```diff
@@ -1,34 +1,41 @@
 <link rel="import" href="../polymer/polymer.html">
 
 <script src="https://s.brightspace.com/lib/siren-parser/6.0.0/siren-parser.js"></script>
-<script src="https://s.brightspace.com/lib/d2lfetch/1.5.3/d2lfetch.js"></script>
+<script src="https://s.brightspace.com/lib/d2lfetch/1.2.0/d2lfetch.js"></script>
```
Comment (ChrisKraljevicD2L) [inline d2l-fetch-siren-entity-behavior.html]: Why the rollback? What was added between `1.2.0` and `1.5.3` that is no longer desired?

Comment (ryantmer) [reply d2l-fetch-siren-entity-behavior.html]: In my `deumdify`ing, I accidentally introduced a bug unfortunately, which wipes out the previous `window.d2lfetch` - fixed in 1.7.0 though.

Comment (awikkerink) [reply d2l-fetch-siren-entity-behavior.html]: We should have a d2l-fetch.html that does the script src for it's current version.

Then we can let bower do dependency resolution for us

Comment (ryantmer) [reply d2l-fetch-siren-entity-behavior.html]: [Yep!](https://github.com/Brightspace/d2l-fetch/blob/master/d2l-fetch.html) :)

Comment (CodeBaboon) [reply d2l-fetch-siren-entity-behavior.html]: Yeah, once we get recent grades into a settled state I plan to update it and the various child projects to use the `.html` version. So for now I'll stick with 1.2.0 here and then move to the new version when I make that change. Sound good?

**Pull request merged**